src/app/profile/payment-methods/payment-method/forms/add-payment-method/modal.tpl.html

```diff
@@ -0,0 +1,87 @@
+<div class="modal-content give-modal-content">
```

Comment:
@adammeyer , you are right. The reason why I did a duplicate 'modal-content' div is because of the styles. Modal form styles are attached to '.modal-content.give-modal-content'. I couldn't find a way to append a class to the 'modal-content' div. Here https://angular-ui.github.io/bootstrap/#/modal I have found a way to add class to a parent modal container or a backdrop, but not 'modal-content'. So I hacked it. It is ugly but I am not sure how else to go around it. Im open for suggestions though =)